export function trycatch() {
  try {
    console.log("Hello World")
  } catch (e) {
    console.error(e)
  }
}
